xsnap/sources/xsnap.c

```diff
@@ -36,37 +36,33 @@ extern void xs_base64_encode(xsMachine *the);
 extern void xs_base64_decode(xsMachine *the);
 extern void modInstallBase64(xsMachine *the);
 
-#define mxSnapshotCallbackCount 23
+#define mxSnapshotCallbackCount 21
 xsCallback gxSnapshotCallbacks[mxSnapshotCallbackCount] = {
 	xs_issueCommand, // 0
 	xs_print, // 1
-	xs_gc, // 2
-
-	xs_clearTimer, // 3
-	xs_setImmediate, // 4
-	xs_setInterval, // 5
-	xs_setTimeout, // 6
```

Comment:
are we removing timers deliberately?

Reply:
Yes. But, we're fine either way. They were put there initially because we thought that was useful for xsnap. That doesn't seem to be the case. If that's correct, better to leave them out. Your choice.